extensions/blocks/mailchimp/mailchimp.php

```diff
@@ -86,3 +89,20 @@ function jetpack_mailchimp_block_load_assets( $attr ) {
 	$html = ob_get_clean();
 	return $html;
 }
+
+/**
+ * Mailchimp connection/list selection verification.
+ *
+ * @return boolean
+ */
+function jetpack_mailchimp_verify_connection() {
+	$option = get_option( 'jetpack_mailchimp' );
```

Comment:
Should we use the `Jetpack_Options` class here?

```suggestion
	$option = Jetpack_Options::get_option( 'mailchimp' );
```